nylas/client/client.py

```diff
@@ -287,7 +287,7 @@ def free_busy(self, emails, start_at, end_at):
             "start_time": start_time,
             "end_time": end_time,
         }
-        resp = self._request(HttpMethod.POST, url, json=data)
+        resp = self._request(HttpMethod.POST, url, json=data, cls=Calendar)
```

Comment:
just to clarify this for me, the class knows what scopes are required from the token? Or is it something completely different?

Reply:
@dantonpimentel no, this is just for the function to notify the request function to use Bearer authentication as it's part of the Calendar API